### ISSUE TITLE
bug: release-please config not used, docs incorrect

### ISSUE THREAD

**Issue by justinthelaw** [googleapis/release-please#2232]
**What you're trying to do**

**_TLDR:_** the release-please workflow does not properly use the `release-please-config.json` regardless of file location or workflow YAML.

I am trying to do the `simple` release method on a repository that has `pyproject` and `npm` maintained dependencies, as well as helm charts and Zarf/UDS packages for kubernetes deployments. I am using the technique where I point the workflow at the `.release-please-manifest.json` and `release-please-config.json` so that it knows what files to modify the version in.

When I switch to node or python, it properly modifies the correct package management files. However, I am trying to use a multi-language/package manager repository in some cases.

**What code you've already tried**

I've tried several variations of the PR/workflow below (placing files in different locations or not explicitly adding them to the workflow file), but the release-please workflow still fails to update the proper files in accordance with my configuration file.

Latest release PR for reference: https://github.com/justinthelaw/repository-template/pull/64
Latest workflow YAML: https://github.com/justinthelaw/repository-template/blob/main/.github/workflows/release-please.yaml
Latest configration file: https://github.com/justinthelaw/repository-template/blob/main/release-please-config.json

**Any error messages you're getting**

None, for the workflow I pointed out above. HOWEVER...

If I try the legacy method of using release-please (mentioned in the [docs](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#using-with-github-actions-and-lerna) here), it fails because that's not a possible option in the YAML (see snippet below too)

```yaml
name: Run Release Please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v2
        id: release
        with:
          command: manifest <- NOT AN OPTION
          token: ${{secrets.GITHUB_TOKEN}}
          default-branch: main
```

**Comment by chingor13** [comment]
The action docs in release-please is out of date. The action is hosted https://github.com/google-github-actions/release-please-action. The latest version is v4.

**Comment by justinthelaw** [comment]
@chingor13 I am using v4 in my provided YAML for the workflow; however, it is not able to properly modify the files contained in the config.

**Comment by justinthelaw** [comment]
I found the correct way to do this was the following:

**WORKFLOW:**

```yaml
name: Release Please

on:
  push:
    branches:
      - main

jobs:
  release-please:
    runs-on: ubuntu-latest
    permissions:
      contents: write
      pull-requests: write
    steps:
      - name: Create release tag
        id: tag
        uses: google-github-actions/release-please-action@cc61a07e2da466bebbc19b3a7dd01d6aecb20d1e # v4
        with:
          command: manifest # use configs in release-please-config.json

```

**CONFIG:**

```json
{
    "packages": {
      ".": {
        "release-type": "simple",
        "draft": false,
        "changelog-path": "CHANGELOG.md",
        "versioning": "default",
        "extra-files": [
          "zarf.yaml",
          "zarf-config.yaml",
          {
            "type": "yaml",
            "path": "chart/Chart.yaml",
            "jsonpath": "$.appVersion"
          },
          {
            "type": "json",
            "path": "package.json",
            "jsonpath": "$.version"
          },
          {
            "type": "toml",
            "path": "pyproject.toml",
            "jsonpath": "$.project.version"
          }
        ]
      }
    }
  }
```

**MANIFEST:***

```json
{
    ".": "0.6.1"
}

```
